index.js

```diff
@@ -213,6 +214,7 @@ async function run() {
     });
 
     // Get inputs
+    const desiredCount = parseInt(core.getInput('desired-count', { required: false })) || 1;
```

Comment:
Setting `desiredCount` to 1 by default will result in any service that currently has a non-1 `desiredCount` to get reset back to 1. I think the logic we want here is to _only_ set this parameter on the `updateService` call if the workflow has explicitly set this option with a value.

We need to add `desired-count` as an input in [`action.yml`](https://github.com/aws-actions/amazon-ecs-deploy-task-definition/blob/master/action.yml#L6)